tests/ImageSharp.Benchmarks/Codecs/DecodeTiff.cs

```diff
@@ -1,38 +1,94 @@
 // Copyright (c) Six Labors.
 // Licensed under the Apache License, Version 2.0.
 
+//// #define BIG_TESTS
+
 using System.IO;
+
 using BenchmarkDotNet.Attributes;
+
+using SixLabors.ImageSharp.Formats.Experimental.Tiff;
 using SixLabors.ImageSharp.PixelFormats;
 using SixLabors.ImageSharp.Tests;
+
 using SDImage = System.Drawing.Image;
 using SDSize = System.Drawing.Size;
 
 namespace SixLabors.ImageSharp.Benchmarks.Codecs
 {
-    [Config(typeof(Config.ShortClr))]
+    [Config(typeof(Clr))]
     public class DecodeTiff : BenchmarkBase
     {
-        private byte[] tiffBytes;
+        public class Clr : Config.ShortClr
```

Comment:
I'd much rather any configuration classes are kept together. If the column with is an issue make it global.

Reply:
yes, I think it would be useful to increase the column for global

Reply:
Done